### ISSUE TITLE
VoiceOver does not focus on new element after opening Create new employee account

### ISSUE THREAD

**Issue by vanpham-sw** [shopware/platform#7820]
### PHP Version

8.2.25

### Shopware Version

6.7

### Affected area / extension

Extension:Commercial

### Actual behaviour

When using VoiceOver to click Create account and open Create new employee account page, the focus does not automatically shift to the expected new element on the page. Instead, the focus remains on the top of the page, causing confusion and requiring additional navigation steps for users relying on VoiceOver.

https://github.com/user-attachments/assets/8e2237e3-1ae9-4ed2-8a89-516119604e0b

### Expected behaviour

Adjust the page load process to ensure that VoiceOver automatically shifts focus to the designated new element upon opening the page.

### How to reproduce

1. Enable VoiceOver on the device.
2. Open the browser and navigate to the Employees page.
3. Use VoiceOver commands to select Create account that opens a new page.

**Comment by vanpham-sw** [comment]
I changed priority to low since:
- it's the same behavior across Storefront.
- it should be nice to have
